src/Service.Host/client/src/components/PurchaseOrders/PurchaseOrderUtility.js

```diff
@@ -557,7 +568,7 @@ function PurchaseOrderUtility({ creating }) {
                                     <Grid item xs={12}>
                                         <ErrorCard
                                             errorMessage={
-                                                itemError?.details ?? itemError.statusText
+                                                itemError?.details?.error ?? itemError.statusText
```

Comment:
What was the actual problem here? I thought @richardiphillips  hinted at standup that it was just a 500 being returned from an unhandled exception, which we should be catching and handling? + will this change not break error message rendering for every other error that currently renders ok?

Reply:
is it a 500 just because its an unhandled exception and thats the default behaviour? can't we just catch the exception properly and return a nice result?

Reply:
So the purchase order form has never been able to display errors correctly?